tests/BreadcrumbTest.php

```diff
@@ -102,4 +102,19 @@ public function testWithoutMetadata(): void
         $this->assertSame(['foo' => 'bar'], $breadcrumb->getMetadata());
         $this->assertArrayNotHasKey('foo', $newBreadcrumb->getMetadata());
     }
+
+    public function testBreadcrumbSerialized(): void
+    {
+        $type = Breadcrumb::TYPE_USER;
+        $category = 'foo';
+        $data = ['baz' => 'bar'];
+        $breadcrumb = new Breadcrumb(Breadcrumb::LEVEL_INFO, $type, $category, null, $data);
```

Comment:
We should also assert that the level and the message of the breadcrumb are correctly serialized

Reply:
added missing parameters